policy-controller/src/main.rs

```diff
@@ -158,10 +158,23 @@ async fn main() -> Result<()> {
         kubert::index::namespaced(index.clone(), http_routes).instrument(info_span!("httproutes")),
     );
 
+    // Create the lease manager used for trying to claim the status-controller lease.
+    let api = k8s::Api::namespaced(runtime.client(), &control_plane_namespace);
+    // todo: Do we need to use LeaseManager::field_manager here?
+    let lease = kubert::lease::LeaseManager::init(api, status::STATUS_CONTROLLER_NAME).await?;
+    let hostname =
+        std::env::var("HOSTNAME").expect("Failed to fetch `HOSTNAME` environment variable");
```

Comment:
It doesn't look like we can provide a more helpful error message when `HOSTNAME` doesn't exist. Through `clap`, it shows up as `error: one or more required arguments were not provided`. We can keep as is with the `expect` so that we can provide a more helpful error message to the user.

Reply:
i believe the clap feature flag `error-context` _might_ fix that, but (as oliver pointed out) this requires additional deps. let's just continue with using `std::env::var`, which i think is totally fine!